internal/resources/ekscluster/resource_ekscluster.go

```diff
@@ -271,6 +272,69 @@ var vpcSchema = &schema.Schema{
 	},
 }
 
+var addonsConfigSchema = &schema.Schema{
```

Comment:
These should have descriptions

Reply:
@TaylorOno I have added the necessary descriptions